ntoskrnl/cc/view.c

```diff
@@ -1005,22 +1024,10 @@ CcRosGetVacb (
     }
 
     Refs = CcRosVacbGetRefCount(current);
-
-    OldIrql = KeAcquireQueuedSpinLock(LockQueueMasterLock);
-
-    /* Move to the tail of the LRU list */
```

Comment:
Just so that I understand what the original purpose of the code was: objects at the _**tail**_ of the LRU list are actually those that have been the _**most**_-recently used?

Reply:
From my understanding of the code, yes.